.jenkinsci/release-build.groovy

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env groovy
+
+def doReleaseBuild() {
+	def parallelism = env.PARALLELISM
+	if ("arm7" in env.NODE_NAME) {
+		parallelism = 1
```

Comment:
Why?

Reply:
ARMv7 machines we run builds on are only 2GB RAM. Each thread during Iroha build consumes just about that amount of memory.